js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,29 @@
+function duplicateLetters(...args) {
```

Comment:
Why not change the function signature and take string as a parameter directly?

Reply:
Yeah, I agree with you. Will update the code.